packages/ketchup/src/components/kup-tab-bar/kup-tab-bar.tsx

```diff
@@ -322,6 +327,20 @@ export class KupTabBar {
                             </span>
                         ) : null}
                     </span>
+                    {toolbar && (
+                        <FImage
+                            resource="app"
+                            sizeX="16px"
+                            sizeY="16px"
+                            onClick={() => {
+                                // e.stopPropagation(); Attivarlo per non propagare l'evento alla selezione del tab-bar
+                                console.log('CONSOLE BTN TOOLBAR');
```

Comment:
only a console.log for this event?

Reply:
it will be managed in parallel with the new toolbar management with a specific kup-event